src/lib.rs

```diff
@@ -13,7 +13,7 @@ extern crate hyper;
 extern crate router;
 extern crate urlencoded;
 extern crate redis;
-extern crate spaceapi;
+pub extern crate spaceapi;
 
 pub use spaceapi as api;
```

Comment:
Apparently "pub extern crate X as Y" works too: https://github.com/rust-num/num/pull/190 Let's use that instead. I don't care about warnings on older Rust versions than 1.8.